packages/tools/devtools/devtools-core/src/data-visualization/DataVisualization.ts

```diff
@@ -82,6 +103,11 @@ export type VisualizeSharedObject = (
  */
 export type VisualizeChildData = (data: unknown) => Promise<VisualChildNode>;
 
+/**
+ * A visual representation of {@link @fluidframework/shared-object-base#ISharedObject} & {@link DataObject} data.
+ */
+export type VisualObject = ISharedObject | PureDataObject;
```

Comment:
Nit: I'm not sure how I feel about this naming. This type doesn't represent something _visual_, it represents something we can _visualize_. How about `VisualizableFluidObject`?